scripts/e2e.ts

```diff
@@ -105,17 +105,20 @@ async function runTest() {
       console.log('No tests to run');
     } else if (selectedProjects) {
       execSync(
-        `yarn nx run-many --target=e2e --projects=${selectedProjects} ${testNamePattern}`,
+        `node -max-old-space-size=4096 ./node_modules/.bin/nx run-many --target=e2e --projects=${selectedProjects} ${testNamePattern}`,
```

Comment:
I think this needs double dashes, ie `--max-old-space-size` 🤔 But I think it better to pass it as environment variable on CircleCI, as suggested [here](https://support.circleci.com/hc/en-us/articles/360009208393-How-can-I-increase-the-max-memory-for-Node-)

Reply:
yep those should be double `--`. Probably copy & paste error from the other PR I have...but let me double check it was this that solved the issue then or the `resource_class` I had also tried before.

yeah I agree, using a ENV variable would be even better 👍 let me check